compiler/scripts/extract-and-check

```diff
@@ -2,24 +2,28 @@
 
 set -e
 
-DIR=$(mktemp -d jasminXXXXXX)
+mkdir -p test_extraction
+DIR=$(mktemp -d -p test_extraction jasminXXXXXX)
 EC=${DIR}/jasmin.ec
 CT=${DIR}/jasmin_ct.ec
 
-if [ "$1" = "arm" ]
-then
+if [ "$1" = "arm" ]; then
 	shift
 	ARCH="-arch arm-m4"
 else
 	ARCH="-arch x86-64"
 fi
 
-trap "rm -r ${DIR}" EXIT
+# trap "rm -r ${DIR}" EXIT
+
+ECLIB=$(dirname $0)/../../eclib
 
 set -x
 
-$(dirname $0)/../jasminc ${ARCH} -oecarray ${DIR} -oec ${EC} "$@"
-$(dirname $0)/../jasminc ${ARCH} -oecarray ${DIR} -CT -oec ${CT} "$@"
+echo $(dirname $0)/../jasminc ${ARCH} -oecarray ${DIR} -oec ${EC} "$2"
```

Comment:
'$2' seems wrong here.

CI says: `Error: File  not found.`

Reply:
The issue was coming from the `shift` above, from an incorrect merge.